app/api-v1/api-doc.js

```diff
@@ -557,13 +588,7 @@ const apiDoc = {
         allOf: [{ $ref: '#/components/schemas/ChainAction' }, { $ref: '#/components/schemas/NewPartMetadataUpdate' }],
       },
     },
-    securitySchemes: {
-      bearerAuth: {
-        type: 'http',
-        scheme: 'bearer',
-        bearerFormat: 'JWT',
-      },
-    },
+    ...securitySchemes,
```

Comment:
I think this is fine. We will likely merge docs together at some point and appliy security schemas then